Python Flask/hello_world.py

```diff
@@ -0,0 +1,10 @@
+#import the Flask class
+from flask import Flask
+
+#create an instance of Flask class
+app = Flask(__name__)
+
+@app.route('/')
+#function that displays 'Hello World'
+def hello_world():
+    return 'Hello, World!'
```

Comment:
This attempt is quite good. But we forgot one thing. To start the server. That part of code would look something like this:
 `if __name__ == '__main__':
        app.run(debug=True)`

Basically we are calling run on app which is an instance of class Flask. And debug=True means we want the debugger to print logs on terminal. You could also set it to false.

Reply:
Impressive comments showing you actually went in depths of what is being imported and what is what. Kudos to that. Great job! 👍 
[Optional] If we could have a comment about what ` __name__`  is , it would be great :)